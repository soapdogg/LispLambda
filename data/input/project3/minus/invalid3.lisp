(MINUS (CONS 34 20) 23)