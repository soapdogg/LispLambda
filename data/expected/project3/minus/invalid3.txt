Error! Parameter at position: 1 of function MINUS is not numeric!    Actual: (34 . 20)
